### DISCUSSION TITLE
Sostituzione oraria

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#430] [Idee e proposte]
Sostituzione oraria firmando però la materia di sostituzione così che gli studenti possano vedere che si sta sostituendo storia matematica…. Come in axios dove prima di firmare avviene la scelta del docente e materia da sostituire.

**Comment by sempliciniluigi89** [comment]
L'abbiamo già proposto qualche anno fa



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
11/23/22,
08:15:41 PM

Il giorno mer 23 nov 2022 alle ore 18:10 Manu ***@***.***> ha
scritto:

**Comment by trinko** [comment]
La segno come richiesta.
